### PR TITLE
cleanup to prepare for ci setup

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: __tests__/addressStorage.test.js [hunk 1]
```diff
@@ -49,26 +49,4 @@ describe("address storage tests", function () {
       done();
     });
   });
-
-  it("will not switch address if address switching disabled", async () => {
```
Comment (ljharb) [inline __tests__/addressStorage.test.js:53]: if the test should be passing but isn’t, then this should use it.skip instead

**Review by ljharb — APPROVED** [review]

File: __tests__/addressStorage.test.js (hunk 1 above)
Comment (bangtoven) [reply __tests__/addressStorage.test.js:53]: turned out the test was not valid.

**Review by ytarsha — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for ytarsha @ 2022-03-23 05:48:18 UTC
User must have write permissions to review

**Review by Mitumaru — COMMENTED** [review]
OPLE: 0xCb4A7569a61300C50Cf80A2be16329AD9F

**Comment by mikehunter1990** [comment]
0xCb4A7569a61300C50Cf80A2be16329AD9F what this

**Pull request merged**